src/aspire/image/image.py

```diff
@@ -465,7 +465,7 @@ def _im_translate(self, shifts):
             xp.asarray(np.ceil(np.arange(-L / 2, L / 2, dtype=self.dtype)))
         )
         grid_1d = xp.asnumpy(grid_shifted) * 2 * np.pi / L
-        om_x, om_y = np.meshgrid(grid_1d, grid_1d, indexing="ij")
+        om_x, om_y = np.meshgrid(grid_1d, grid_1d, indexing="xy")
```

Comment:
Lets add a comment for the next developer. Thanks

Reply:
Sure